README.md

```diff
@@ -0,0 +1,56 @@
+# OpenSAFELY Command Line Interface
+
+This tool is part of the [OpenSAFELY](https://www.opensafely.org)
+project.
+
+To use it you will need:
+ * Python version 3.7 or above
```

Comment:
We had to remove 3.8-specific syntax from the original cohortextractor because people didn't have that installed locally and we were trying to minimise friction. I guess eventually we'll be shipping our own binary and this will be irrelevant, but right now I want to make sure that this works in whatever environment people have currently been using.

Reply:
👌 

Not relevant to this PR I think but probably best to write that down somewhere?